apps/dcos_l4lb/src/dcos_l4lb_mgr.erl

```diff
@@ -28,6 +28,7 @@
     last_received_vips = [],
```

Comment:
It's not possible to cherry-pick any changes in this module due to heavy refactoring that was done in 1.12 release. so it's new code.